internal/isoutil/isoutil.go

```diff
@@ -228,3 +232,64 @@ func (h *installerHandler) VolumeIdentifier() (string, error) {
 
 	return strings.TrimSpace(string(volumeId)), nil
 }
+
+func (h *installerHandler) GetFileLocation(filePath string) (int64, error) {
+	fs, err := h.isoFS()
+	if err != nil {
+		return 0, err
+	}
+
+	isoFile, err := fs.OpenFile(filePath, os.O_RDONLY)
+	if err != nil {
+		return 0, errors.Wrapf(err, "Failed to open file %s", filePath)
+	}
+
+	myFile := isoFile.(*iso9660.File)
+	defaultSectorSize := int64(2 * 1024)
+	offset := int64(myFile.Location()) * defaultSectorSize
+	return offset, nil
+}
+
+func (h *installerHandler) Copy(filePath string) error {
+	fsFile, err := os.OpenFile(h.isoPath, os.O_RDONLY, 0o664)
+	if err != nil {
+		return err
+	}
+
+	osFile, err := os.Create(filePath)
+	if err != nil {
+		return err
+	}
+
+	_, err = io.Copy(osFile, fsFile)
+	if err != nil {
+		osFile.Close()
+		return err
+	}
+
+	if err := osFile.Close(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (h *installerHandler) Write(location int64, data []byte, filePath string) error {
```

Comment:
I'm not sure this belongs here.
At the very least it shouldn't be a method on the handler as it doesn't use any of that data.

Reply:
Also I think this whole method is a one-liner if we open the iso as a `os.File`
`func (f *File) WriteAt(b []byte, off int64) (n int, err error)`

So I would definitely say remove this and use `WriteAt` from the caller.

Reply:
Cool, seems to do the trick :)